iOS/Flows/Conversation/Cells/ConversationMessageCellDatasource.swift

```diff
@@ -94,21 +81,18 @@ extension ConversationMessageCellDataSource {
 
 extension ConversationMessageCellDataSource: TimeMachineCollectionViewLayoutDataSource {
 
-    func getConversation(forItemAt indexPath: IndexPath) -> Conversation? {
-        guard let item = self.itemIdentifier(for: indexPath) else { return nil }
-
-        return ChatClient.shared.channelController(for: item.channelID).conversation
-    }
-
-    func getMessage(forItemAt indexPath: IndexPath) -> Messageable? {
```

Comment:
No need to get the conversation to make the time machine work. Deleting this delegate function. 

I also made the time machine less message specific. It just asks for a "TimeMachineLayoutItem".